ee/clickhouse/queries/trends/person.py

```diff
@@ -95,10 +95,11 @@ def actor_query(self, limit_actors: Optional[bool] = True) -> Tuple[str, Dict]:
             filter=self._filter,
             team=self._team,
             entity=self.entity,
-            should_join_distinct_ids=not self.is_aggregating_by_groups,
-            should_join_persons=not self.is_aggregating_by_groups,
```

Comment:
Person join only needs to happen if we have person properties to handle, not every time. (Notice snapshot tests being updated, where the person join was unnecessary)